app/src/main/java/com/example/fittracker/mvp/view/MainScreenView.java

```diff
@@ -77,17 +79,16 @@ public void onBackPressed() {
 
     @Override
     public void onLogOutPressed() {
-        SharedPreferences.Editor editor = activity.get().
-                getSharedPreferences(ConstantUtils.USER_PREFERENCES, Context.MODE_PRIVATE).edit();
-        editor.putInt(ConstantUtils.USER_PREFERENCES_ID, ConstantUtils.NEGATIVE_ONE);
-        editor.apply();
         activity.get().finish();
     }
 
     @Override
     public boolean locationPermissionGranted() {
-        return (ActivityCompat.checkSelfPermission(activity.get(),
-                Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED);
+        boolean isGranted = ((ContextCompat.checkSelfPermission(activity.get(),
+                Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED)
+                && (ContextCompat.checkSelfPermission(activity.get(),
+                Manifest.permission.ACCESS_COARSE_LOCATION) == PackageManager.PERMISSION_GRANTED));
+        return isGranted;
```

Comment:
isGranted boolean is not necessary here, please remove it

Reply:
Done